### DISCUSSION TITLE
Call actions and workflow of private repository

### DISCUSSION THREAD

**Discussion by mwagenma-maersk** [community/community#16243] [Enterprise] [answered]
It is possible to call actions of private repositories. However, calling workflows of private repositories is not possible. Are there any plans on changing this or any good reasons against it?

**Comment by mwagenma-maersk** [comment, marked as answer]
This shoula already be possible as is stated here: https://docs.github.com/en/enterprise-cloud@latest/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#allowing-access-to-components-in-an-internal-repository.

We will need to look into why this does not work in our organization ourselves.

**Comment by peter900932** [comment]
Allowing workflow calls from private repositories could improve automation flexibility, but concerns about security, access control, and unintended exposure of sensitive data must be addressed. Ensuring proper permissions and visibility for private workflows is key for this change. You can also get [more info](https://koziolthomseye.com/) on relevent sites.

**Comment by saboorali76** [comment]
The Cookout Menu offers detailed insights into Cook Out's food lineup, making it easier for customers to explore menu items before placing an order. From classic trays and sandwiches to refreshing beverages and desserts, [the website](https://thecookoutsmenu.us/) delivers helpful information that enhances meal planning and restaurant visits.

**Reply by v-coder247** [reply]
Hi @saboorali76, thanks for sharing with the community! 


Unfortunately, we currently do not allow self-promotion, advertising, or solicitation in Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
